package.json

```diff
@@ -141,7 +141,8 @@
     "react-day-picker": "^7.2.4",
     "react-material-icon-svg": "1.7.0",
     "react-popper": "^1.3.3",
-    "react-transition-group": "^2.4.0"
+    "react-transition-group": "^2.4.0",
+    "resize-observer-polyfill": "^1.5.1"
```

Comment:
https://github.com/que-etc/resize-observer-polyfill 
It has really good browsers support.